Source/DataSources/EntityView.js

```diff
@@ -162,18 +173,17 @@ define([
                 Transforms.eastNorthUpToFixedFrame(cartesian, ellipsoid, transform);
             }
 
-            var offset;
-            if ((mode === SceneMode.SCENE2D && that._offset2D.range === 0.0) || (mode !== SceneMode.SCENE2D  && Cartesian3.equals(that._offset3D, Cartesian3.ZERO))) {
-                offset = undefined;
-            } else {
-                offset = mode === SceneMode.SCENE2D ? that._offset2D : that._offset3D;
-            }
+            camera._setTransform(transform);
```

Comment:
@bagnell and I talked about this private access, decided to let it slide since `EntityView` will get rewritten soon.